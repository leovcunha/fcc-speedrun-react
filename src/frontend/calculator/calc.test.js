import React from "react";
import renderer from "react-test-renderer";
import Calculator from "../calculator";

describe("calculator", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<Calculator/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
