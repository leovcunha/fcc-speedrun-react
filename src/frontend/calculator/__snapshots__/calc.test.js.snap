// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calculator renders without crashing 1`] = `
<div
  className="calculator jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Calculator
  </div>
  <hr
    className="p-5"
  />
  <div
    className="calc row justify-content-center"
  >
    <table>
      <tbody>
        <tr>
          <td
            className="display2"
            colSpan="4"
          >
            
          </td>
        </tr>
        <tr>
          <td
            className="display1"
            colSpan="4"
          >
            0
          </td>
        </tr>
        <tr>
          <td
            colSpan="2"
            onClick={[Function]}
          >
            C
          </td>
          <td
            onClick={[Function]}
          >
            âˆš
          </td>
          <td
            onClick={[Function]}
          >
            %
          </td>
        </tr>
        <tr>
          <td
            onClick={[Function]}
          >
            1
          </td>
          <td
            onClick={[Function]}
          >
            2
          </td>
          <td
            onClick={[Function]}
          >
            3
          </td>
          <td
            onClick={[Function]}
          >
            +
          </td>
        </tr>
        <tr>
          <td
            onClick={[Function]}
          >
            4
          </td>
          <td
            onClick={[Function]}
          >
            5
          </td>
          <td
            onClick={[Function]}
          >
            6
          </td>
          <td
            onClick={[Function]}
          >
            -
          </td>
        </tr>
        <tr>
          <td
            onClick={[Function]}
          >
            7
          </td>
          <td
            onClick={[Function]}
          >
            8
          </td>
          <td
            onClick={[Function]}
          >
            9
          </td>
          <td
            onClick={[Function]}
          >
            x
          </td>
        </tr>
        <tr>
          <td
            onClick={[Function]}
          >
            0
          </td>
          <td
            onClick={[Function]}
          >
            .
          </td>
          <td
            onClick={[Function]}
          >
            =
          </td>
          <td
            onClick={[Function]}
          >
            /
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
